docs/modules/ROOT/examples/go/example.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hazelcast/hazelcast-go-client"
+	"github.com/hazelcast/hazelcast-go-client/cluster"
+)
+
+func main() {
+	var clusterConfig cluster.Config
+	clusterConfig.Network.SetAddresses("172.18.254.1")
```

Comment:
Could you replace the ip for all client samples with place-holder like <LOAD_BALANCER_IP>?

it would be great if you can provide a step to fetch this IP.

Reply:
Sorry, I missed that 🤦‍♂️ 

For the IP change, you can follow other tutorials' way:
https://github.com/hazelcast-guides/hazelcast-platform-operator-expose-externally/blob/main/go/main.go#L14